### DISCUSSION TITLE
How to concatenate paths in service volumes property

### DISCUSSION THREAD

**Discussion by bxavi** [community/community#48070] [Actions]
**Services in Github Actions**

I would like to concatenate some strings to mount a volume in postgres container. I would like to use syntax like this:

`volumes:`
`- ${{ github.workspace }}/pg:/pg`

Which does not work. The folder appears empty in a later step when queried like this:

`- run: docker exec containername ls pg`

I have tried creating an environment variable but I dont think that this can be referenced from a service definition.

Any help appreciated

**Comment by airtower-luna** [comment]
> The folder appears empty in a later step when queried like this:

What do you expect to be in the folder? By the time a service container is started there shouldn't be anything in the workspace. Or in general, what are you trying to achieve?

> I have tried creating an environment variable but I dont think that this can be referenced from a service definition.

That's correct, environment variables are [only available inside job steps](https://docs.github.com/en/actions/learn-github-actions/contexts#env-context).
